lib/puppet.rb

```diff
@@ -33,6 +33,10 @@
   def _(msg)
     msg
   end
+
+  def n_(msg_singular, msg_plural, count)
+    return count == 1 ? msg_singular : msg_plural
```

Comment:
Yeah it does seem good to do in general. Sigh, gotta love Ruby >.> I'm happy to change the signature and then just keep the functionality as it is now, i.e. grab the first two args of the list and return the right one based on `count`. I'll also add the block.